### PR TITLE
{devel}[foss/2022a] gemelli v0.0.9 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node366.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/4f7c7a4f0e503e264a84ef677a6b3262 for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/g/gemelli/gemelli-0.0.9-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+]
+
+exts_list = [
+    ('tax2tree', '1.1', {
+        'modulename': 't2t',
+        'checksums': ['f1d28fb99c41bb10cc3c06bfd232891b0ad713f674a5c2ac2991dd99ed512da1'],
+    }),
+    ('iow', '1.0.6', {
+        'modulename': 'bp',
+        'preinstallopts': """sed -i "/'nose /d" setup.py && """,
```
Comment (Micket) [inline easybuild/easyconfigs/g/gemelli/gemelli-0.0.9-foss-2022a.eb:30]: Why not include nose and 
```
'runtest': 'nosetests'
```
(or whatever the syntax is, i think this works)

Comment (smoors) [reply easybuild/easyconfigs/g/gemelli/gemelli-0.0.9-foss-2022a.eb:30]: nose is not maintained for a long time and not compatible with python3.10
https://github.com/nose-devs/nose/issues/1099

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 525.105.17, Python 3.9.16
See https://gist.github.com/SebastianAchilles/bf6c9e718ed516a2d95659bbf78b2f87 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-opensuse-155 - Linux openSUSE Leap 15.5, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.15
See https://gist.github.com/SebastianAchilles/484eef2aa7087f1ee194591ae1fa97ed for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @smoors!

**Pull request merged**